README.md

```diff
@@ -48,7 +48,7 @@ There are a variety of image tags available:
 (Replace empty config strings with correct values.)
 
 ```bash
-docker run -it -v "$(pwd)/data":/data -e DISCORD_TOKEN='' -e SPOTIFY_CLIENT_ID='' -e SPOTIFY_CLIENT_SECRET='' -e YOUTUBE_API_KEY='' codetheweb/muse:latest
+docker run -it -v "$(pwd)/data":/data -e DISCORD_TOKEN='' -e SPOTIFY_CLIENT_ID='' -e SPOTIFY_CLIENT_SECRET='' -e YOUTUBE_API_KEY='' -e NODE_ENV='development' codetheweb/muse:latest
```

Comment:
Shouldn't this be `NODE_ENV=production` instead?

We should set this directly in the Dockerfile instead of making the consumer do it.

Reply:
Ok, I moved the variable directly  in the Dockerfile, but I think it's better to default to develop and let the user specify when he wants to deploy in production? I've commented the NODE_ENV override in docker-compose so they can just uncomment it to set in production. I don't know in Node, but in C# the projects default to develop, maybe it's different in Node...
Let me know what's better

Reply:
Hmm, I think you might be right.

The main downside of global commands is that it can take up to an hour for updates to propagate, so keep `NODE_ENV` defaulting to empty / development for now I guess and users can switch to production if they happen to be running the bot in a very large number of guilds for some reason.